Storage/ThirdParty/SwiftData.swift

```diff
@@ -840,13 +831,11 @@ open class ConcreteSQLiteDBConnection: SQLiteDBConnection {
                 
                 log.debug("Finished moving database \(baseFilename) successfully.")
             } catch let error as NSError {
-                log.error("Unable to move \(baseFilename) to another location. \(error)")
-                SentryIntegration.shared.sendWithStacktrace(message: "Unable to move \(baseFilename) to another location. \(error)", tag: "SwiftData", severity: .error)
+                Sentry.shared.sendWithStacktrace(message: "Unable to move db to another location", tag: SentryTag.swiftData, severity: .error, description: "\(baseFilename) to another location. \(error)")
```

Comment:
Change to: `description: "DB file '\(baseFilename)'. \(error.localizedDescription)"`